UPGRADE-6.1.md

```diff
@@ -13,6 +13,11 @@ Console
 
  * Deprecate `Command::$defaultName` and `Command::$defaultDescription`, use the `AsCommand` attribute instead
 
+FrameworkBundle
+---------------
+
+ * Deprecate the `reset_on_message` config option. It can be set to `true` only and does nothing now
```

Comment:
Can you mention the alternative?

Reply:
What is the alternative? The `reset_on_message` does nothing already. It was nedded to indicate the changing of the default behaviour in 6.0. 

The original plan: https://github.com/symfony/symfony/pull/43133#issuecomment-927140162
Future details: https://github.com/symfony/symfony/pull/43203#issuecomment-954750050

Reply:
Should I add the info: "To prevent services resetting after each message you can set the "--no-reset" option in "messenger:consume" command?"

Reply:
Sounds good to me @upyx

Reply:
@chalasr @fabpot 
Done